payments-ui-core/src/test/java/com/stripe/android/ui/core/elements/CardNumberControllerTest.kt

```diff
@@ -11,7 +13,9 @@ import org.robolectric.RobolectricTestRunner
 
 @RunWith(RobolectricTestRunner::class)
 internal class CardNumberControllerTest {
```

Comment:
We should add some tests for the metadata service.  I think this might motivate the need for a helper class, as unit testing might be difficult without it.